### PR TITLE
Active dedi tab fix

### PR DESCRIPTION
**Pull request by J-Wood105**
This basically just makes the selection of a dedi update the info on the right, as well as updating the url.

### CODE REVIEW

File: dedi-mgmt.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var proxy_prefix = "http://" + proxy_ip + ":" + proxy_port;
 var countdownDefault = 10;
 var apiFailMax = 5;
 
-Vue.createApp({
+const DediApp = Vue.createApp({
```
Comment (thetrackavl) [inline dedi-mgmt.js:9]: no problem with this except I don't know the implications...anything significant with adding cost or just best practice?

File: dedi-mgmt.js [hunk 2]
```diff
@@ -151,14 +151,15 @@ Vue.createApp({
 				// console.log("starting dedi: " + dedi.dediId + " " + dedi.apiFailSession);
 				try {
 					// console.log(dedis[index])
-					app.manageData(
```
Comment (thetrackavl) [inline dedi-mgmt.js:154]: looks like we have a difference in our formatters...we should sync these somehow

File: dedi-mgmt.js [hunk 3]
```diff
@@ -295,16 +300,13 @@ Vue.createApp({
...
-						Math.random() * countdown
-					);
+					dataObj[countdownKey] = Math.floor(Math.random() * countdown);
 				}
 				// return dataObj;
 			}
 			return dataObj;
 		},
 		errorDediSession: function (dedi) {
-			dedi.serverName = "error";
```
Comment (thetrackavl) [inline dedi-mgmt.js:307]: why was this removed?

File: dedi-mgmt.html [hunk 4]
```diff
@@ -30,13 +30,14 @@
...
+							v-bind:key="dedi.serverName"
 							v-on:click="activateTab(dedi.serverName)"
 							v-bind:class="[ activeTab == dedi.serverName ? 'active' : '' ]"
 							class="nav nav-link"
 							style="cursor:pointer"
-							id="dedi.serverName"
+							v-bind:id="dedi.serverName"
 							data-toggle="tab"
-							href="#dedi.serverName"
+                                                        v-bind:href="`#${dedi.serverName}`"
```
Comment (thetrackavl) [inline dedi-mgmt.html:40]: ahh that makes sense

File: proxy.js [hunk 5]
```diff
@@ -2,7 +2,7 @@ const express = require("express");
 const request = require("request");
 const axios = require("axios");
 
-var default_api_ip = "192.168.2.102";
+var default_api_ip = process.env.API_IP || "192.168.2.102";
```
Comment (thetrackavl) [inline proxy.js:5]: is this just setting up to pull info from env like credentials?

File: dedi-mgmt.js (hunk 2 above)
Comment (J-Wood105) [reply dedi-mgmt.js:154]: yeah, you can send me your vscode json configuration for the prettier plugin we can go that route, or introduce something like eslint as an automatic formatter.

File: proxy.js (hunk 5 above)
Comment (J-Wood105) [reply proxy.js:5]: yeah, this is for when I finally have a better test environment server setup. That way I can just have an environemnt variable set instead of having to remember to change this back and forth. We can try going about it another way if you prefer.

File: dedi-mgmt.js (hunk 3 above)
Comment (J-Wood105) [reply dedi-mgmt.js:307]: may bad on that. Since I'm still setting up a test/dev environment I was doing some changes to make sure things showed as if they succedded. I missed changing this back. I'll update the branch.

Comment (thetrackavl) [reply dedi-mgmt.js:307]: 👍

File: dedi-mgmt.js (hunk 2 above)
Comment (thetrackavl) [reply dedi-mgmt.js:154]: 👍  I'll email my prettier conf if that works

File: proxy.js (hunk 5 above)
Comment (thetrackavl) [reply proxy.js:5]: nope, fine with it, just want to make sure I understand!

**Review by thetrackavl — APPROVED** [review]
